llvm-hs/src/Control/Monad/AnyCont/Class.hs

```diff
@@ -24,7 +24,7 @@ instance MonadTransAnyCont b m => MonadAnyCont b (AnyContT m) where
   anyContToM c = AnyCont.anyContT (liftAnyCont c)
 
 instance Monad m => ScopeAnyCont (AnyContT m) where
-  scopeAnyCont = lift . flip AnyCont.runAnyContT return
+  scopeAnyCont a = lift (((\cont -> AnyCont.runAnyContT cont return)) a)
```

Comment:
Am I reading this incorrectly, or are there unnecessary parens here, around the `\cont -> ... return` part? I think the rhs expr is equivalent to `lift $ AnyCont.runAnyContT a return`,

Reply:
I've done theses changes with search and replace, so yeah, that's totally possible.